core/kits/documents/tabs/settings-layout.php

```diff
@@ -69,11 +71,21 @@ protected function register_tab_controls() {
 						'max' => 1500,
 						'step' => 10,
 					],
-				],
-				'description' => esc_html__( 'Sets the default width of the content area (Default: 1140)', 'elementor' ),
-				'selectors' => [
-					'.elementor-section.elementor-section-boxed > .elementor-container' => 'max-width: {{SIZE}}{{UNIT}}',
-					'.e-con' => '--container-max-width: {{SIZE}}{{UNIT}}',
+					'%' => [
+						'min' => 50,
+						'max' => 100,
+						'step' => 1,
+					],
+					'vw' => [
+						'min' => 50,
+						'max' => 100,
+						'step' => 1,
+					],
+					'description' => esc_html__( 'Sets the default width of the content area (Default: 1140px)', 'elementor' ),
+					'selectors' => [
+						'.elementor-section.elementor-section-boxed > .elementor-container' => 'max-width: {{SIZE}}{{UNIT}}',
+						'.e-con' => '--container-max-width: {{SIZE}}{{UNIT}}',
+					],
```

Comment:
You moved the `description` and the `selectors` inside the `range`.

Reply:
Done.